package.json

```diff
@@ -1,25 +1,24 @@
 {
   "name": "prismarine.js.org",
   "version": "1.0.0",
+  "description": "PrismarineJS website build with Next.js",
   "private": true,
   "homepage": "https://prismarinejs.github.io/",
   "dependencies": {
-    "bootstrap": "^5.0.1",
-    "react": "^17.0.1",
-    "react-dom": "^17.0.1",
-    "react-router-dom": "^5.2.0",
-    "react-scripts": "^3.4.1"
+    "next": "latest",
```

Comment:
That's a bad practice, use Carret versions

Reply:
Yep I will change this